packages/mutation-testing-elements/src/components/mutation-test-report-file.ts

```diff
@@ -0,0 +1,94 @@
+import { LitElement, html, property, customElement, css, PropertyValues } from 'lit-element';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import hljs from 'highlight.js/lib/highlight';
```

Comment:
Adding to it, in my opinion the default highlight.js theme is quite ugly. Perhaps we could use a different one. [There are quite a few nice ones](https://highlightjs.org/static/demo/). On stryker-mutator.io we use the GitHub theme. Atom One Light is also nice. Or Monokai or Atom One Dark would be my suggestions if we want to have a dark theme

Reply:
> we use the GitHub theme. Atom One Light is also nice. Or Monokai or Atom One Dark would be my suggestions if we want to have a dark theme

So GitHub? 😅

Reply:
I mean, if you want to. I personally prefer some of the others though. Choosing the theme shouldn't be based on what's already on the website, as that can also be changed easily

Reply:
I'll add "github" for now. We can later make it dynamic by adding it as a property (`<mutation-test-report-app highlight-theme="github"></mutation-test-report-app>`)